src/GridList/GridTile.js

```diff
@@ -0,0 +1,154 @@
+// @flow weak
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { createStyleSheet } from 'jss-theme-reactor';
+import customPropTypes from '../utils/customPropTypes';
+
+export const styleSheet = createStyleSheet('MuiGridTile', (theme) => {
+  return {
+    root: {
+      position: 'relative',
+      display: 'block',
+      height: '100%',
+      overflow: 'hidden',
+      fontFamily: theme.typography.fontFamily,
+    },
+    childImg: {
+      height: '100%',
+      transform: 'translateX(-50%)',
+      position: 'relative',
+      left: '50%',
+      maxWidth: 'none', // Reset to initial value to override docs-site App-frame.
```

Comment:
I think that we should be removing the global style override of the documentation. That's creating an inconsistency. Users might not get the same output. We already have that issue with the Card. A visual regression test isn't ouput the same thing (broken as missing the docs global overrides). 🙈 .